templates/Gemfile.erb

```diff
@@ -12,6 +12,9 @@ gem "oj", "3.5.1"
 <% if !is_v1 %>
 gem "fluent-plugin-secure-forward"
 gem "fluent-plugin-record-reformer"
+gem "fluent-plugin-rewrite-tag-filter", "<2.0.0"
+<% else %>
```

Comment:
I don't think we need these version constraints here, the version constraints seem fine on the gem ...

Reply:
Also it might be nicer to bump the versions up for all the other gems in a separate commit from adding `fluent-plugin-rewrite-tag-filter`, it will make the history cleaner ...

Reply:
> Also it might be nicer to bump the versions up for all the other gems in a separate commit from adding fluent-plugin-rewrite-tag-filter, it will make the history cleaner ...

Reply:
With bundler, correct version is installed because bundler uses different algorithm for resolving version dependency.

You can check the behaviour with following Gemfile.

```
source 'https://rubygems.org/'

#gem 'fluentd', '~> 1' # install rewrite-tag-filter v2
gem 'fluentd', '~> 0.12.0' # install rewrite-tag-filter v1
gem 'fluent-plugin-rewrite-tag-filter'
```